### PR TITLE
feat(arduino-board-files): disable magdeck motor in bootloader

### PR DESCRIPTION
**Pull request by sanni-t**
## overview

The magdeck motor's enable pin is active low. When in the bootloader, this pin is 0 by default, so the motor stays on during a firmware update and is capable of pulling high currents. This sometimes damages the motor.

## changes

- Updated the Opentrons avr bootloader to pull it HIGH.
- Made changes to makefile to separate magdeck bootloader build.
- Bumped up arduino boards version.

(Sorry for the diffs in line endings in some of the files. I'm marking the true code changes in these so they're easier to spot)

### CODE REVIEW

File: arduino-board-files/opentrons-1.3.0/bootloaders/caterina/Caterina.c [hunk 1]
```diff
@@ -171,19 +171,22 @@ void SetupHardware(void)
 	/* Relocate the interrupt vector table to the bootloader section */
 	MCUCR = (1 << IVCE);
 	MCUCR = (1 << IVSEL);
-	
+
+  #if defined(OT_MAGDECK_ONLY)
+    OT_MOTOR_DISABLE();
+  #endif
```
Comment (sanni-t) [inline arduino-board-files/opentrons-1.3.0/bootloaders/caterina/Caterina.c:177]: 👆

File: arduino-board-files/opentrons-1.3.0/bootloaders/caterina/Caterina.h [hunk 2]
```diff
@@ -63,18 +63,19 @@
...
 		/** Eight character bootloader firmware identifier reported to the host when requested */
 		#define SOFTWARE_IDENTIFIER          "CATERINA_Opentrons_Mod"
-		
+
 		#define CPU_PRESCALE(n)	(CLKPR = 0x80, CLKPR = (n))
 		#define LED_SETUP()		DDRE |= (1<<6); DDRB |= (1<<0); DDRD |= (1<<5);
 		#define L_LED_OFF()		PORTE &= ~(1<<6)
 		#define L_LED_ON()		PORTE |= (1<<6)
 		#define L_LED_TOGGLE()	PORTE ^= (1<<6)
+		#define OT_MOTOR_DISABLE()		DDRB |= (1<<6); PORTB |= (1<<6);
```
Comment (sanni-t) [inline arduino-board-files/opentrons-1.3.0/bootloaders/caterina/Caterina.h:72]: 👆

File: arduino-board-files/opentrons-1.3.0/bootloaders/caterina/Descriptors.c [hunk 3]
```diff
@@ -191,19 +191,23 @@ const USB_Descriptor_String_t LanguageString =
...
-	.Header                 = {.Size = USB_STRING_LEN(20), .Type = DTYPE_String},
+	.Header                 = {.Size = USB_STRING_LEN(21), .Type = DTYPE_String},
 
 	#if DEVICE_PID == 0xEE5A
-	.UnicodeString			= L"OT Module Bootloader"
+		#if defined(OT_MAGDECK_ONLY)
+		.UnicodeString			= L"OT Magdeck Bootloader"
+		#else
+		.UnicodeString			= L"OT Module Bootloader"
+		#endif
```
Comment (sanni-t) [inline arduino-board-files/opentrons-1.3.0/bootloaders/caterina/Descriptors.c:201]: 👆

**Review by sfoster1 — APPROVED** [review]
LGTM

**Pull request merged**